### ISSUE TITLE
LFO-CV Capability

### ISSUE THREAD

**Issue by mathDR** [arviz-devs/arviz#603]
Implementation of LFO-CV from https://github.com/paul-buerkner/LFO-CV-paper

**Comment by ahartikainen** [comment]
We already have khat calculation, but I think the method needs refitting the model multiple times.

So we would need to add `loo`-kind function, that does not do refit, but outputs information for the conditions how to select data? And then some other function that summarises the procedure?

Is there a way to create function wrapper that does this automatically with all the libraries we have?

**Comment by OriolAbril** [comment]
I think that the algorithm linked is approximate LFO-CV, using PSIS. It is similar to PSIS LOO-CV, but it takes into account the time dependency. Figure 3 in the paper is probably a good summary.

**Comment by ahartikainen** [comment]
Yes, but it still needs refitting right?

**Comment by OriolAbril** [comment]
Yes, I missed the initial refit reading in diagonel. We can look a little into it and make some experiments though, to see to which extent is refitting necessary. In the paper they say that in many cases one refit (using only the first L points can be enough), thus, if this covers a wide enough range of cases, it may be useful to implement, and require 2 inference data objects as input, the fit on the whole time series and the fit on the first L observations. 

The wrapper is not bad idea either. It would have more applications in addition to this like [reloo](https://www.rdocumentation.org/packages/brms/versions/2.8.0/topics/reloo), but it seem really complicated, and even more to maintain for all the packages.

**Comment by OriolAbril** [comment]
LFO was started in #771

**Comment by mathDR** [comment]
Going to close this as I _think_ #771 took care of it?
